ci-operator/step-registry/openstack-k8s-operators/deploy/openstack-k8s-operators-deploy-commands.sh

```diff
@@ -30,8 +30,18 @@ if [[ "$REF_ORG" != "$ORG" ]]; then
 fi
 SERVICE_NAME=$(echo "${BASE_OP^^}" | sed 's/\(.*\)-OPERATOR/\1/'| sed 's/-/\_/g')
 
-export IMAGE_TAG_BASE=${REGISTRY}/${ORGANIZATION}/${OPENSTACK_OPERATOR}
-export OPENSTACK_OPERATOR_INDEX=${IMAGE_TAG_BASE}-index:${PR_SHA}
+# Copy base operator code to home directory
+cp -r /go/src/github.com/${ORG}/${BASE_OP}/ ${HOME}
+
+if [[ "$SERVICE_NAME" == "INSTALL_YAMLS" ]]; then
+  # when testing install_yamls patch, we can skip build process and
+  #  validate using latest openstack-operator tag
+  export IMAGE_TAG_BASE=quay.io/${ORG}/${OPENSTACK_OPERATOR}
+  export OPENSTACK_OPERATOR_INDEX=${IMAGE_TAG_BASE}-index:latest
+else
+  export IMAGE_TAG_BASE=${REGISTRY}/${ORGANIZATION}/${OPENSTACK_OPERATOR}
+  export OPENSTACK_OPERATOR_INDEX=${IMAGE_TAG_BASE}-index:${PR_SHA}
+fi
```

Comment:
this seems like bad engineering but i must be missing context. Like why do we have ORG vs ORGANIZATION i.e. two variables for the same thing.

Reply:
Yes, same for registry. This needs a rework on var names. ORG was the default ORGANIZATION name, which is also used as registry namespace. So we might need to rename to REGISTRY_ORG or REG_NAMESPACE instead. I can propose a follow up patch for that, instead of this one to mix things, wdyt?

Reply:
Good point marios, Renaming in a follow-up sounds good to me, this patch looks good to me otherwise. I will give a chance to marios to revist doug's comment.

Reply:
yes ok for followup 
thanks for checking and it seems we all agree so great :)